lib/devise-security/models/active_record/old_password.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
-class OldPassword < ApplicationRecord
```

Comment:
This should have a check for Rails 5 and use `ApplicationRecord` if it's supported.

Reply:
Also should have a test